client/java/src/main/java/com/linecorp/centraldogma/client/WatchConstants.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.centraldogma.client;
+
+import java.util.concurrent.TimeUnit;
+
+final class WatchConstants {
+
+    static final long DEFAULT_WATCH_TIMEOUT_MILLIS = TimeUnit.MINUTES.toMillis(1);
+    static final int RECOMMENDED_AWAIT_TIMEOUT_SECONDS = 20;
```

Comment:
How many times can the CD-client retry a request with the default retry policy within 20 seconds? If it's greater than or equal to 3 including the first request, then it seems acceptable.

Reply:
- At t0 + 0s: initial try
- At t0 + 2s: 1st retry
- At t0 + 6s: 2nd retry
- At t0 + 14s: 3rd retry
- At t0 + 30s: 4th retry

i.e. 20s timeout will give a user 4 tries.